### PR TITLE
Handle http 400 errors a little more gracefully

### PR DESCRIPTION
**Pull request by AWare**
OLD:
image

NEW:
image

### CODE REVIEW

**Comment by AWare** [comment]
New text: 
image

File: frontend/app/monitoring/ErrorHandler.scala [hunk 1]
```diff
@@ -43,4 +43,9 @@ class ErrorHandler @Inject() (
 
   override protected def onProdServerError(request: RequestHeader, exception: UsefulException): Future[Result] =
     Future.successful(NoCache(InternalServerError(views.html.error500(exception))))
+
+  override protected def onBadRequest(request: RequestHeader, message: String): Future[Result] = {
+    logServerError(request, new PlayException("Bad request","A very bad request was received!"))
```
Comment (paulbrown1982) [inline frontend/app/monitoring/ErrorHandler.scala]: 'very'?

**Comment by paulbrown1982** [comment]
👍  
Also we have a ticket to do the same in Subscriptions: https://trello.com/c/58tIPCvq/470-play-error-messages-in-prod as raised by @adamnfish

**Comment by AWare** [comment]
Seen in prod :) 
Raised against subs: https://github.com/guardian/subscriptions-frontend/pull/423

**Pull request merged**